lib/GuestManager.php

```diff
@@ -205,16 +210,36 @@ public function sendEmailInvitation(Room $room, Participant $participant): void
 			$subject
 		);
 
-//		if ($pin) {
-//			// FIXME wrap in text
-//			$template->addBodyText($pin);
-//		}
-
 		$template->addBodyButton(
 			$this->l->t('Join »%s«', [$room->getDisplayName('')]),
 			$link
 		);
 
+		if ($pin) {
+			$template->addBodyText(
+				htmlspecialchars($this->l->t('You can also dial-in via phone with the following details')),
+				$subject
+			);
```

Comment:
Shouldn't this be `$template->addBodyText($this->l->t('You can also dial-in via phone with the following details'))`? Otherwise instead of _You can also dial-in via phone with the following details_ [the e-mail subject would be shown in plain text e-mails](https://github.com/nextcloud/server/blob/d9015a8c94bfd71fe484618a06d276701d3bf9ff/lib/private/Mail/EMailTemplate.php#L420-L443) (also if no explicit plain text is given then `htmlspecialchars` is no longer needed, as it is automatically applied by `addBodyText`).

However not that I have not checked a real e-mail, only looked at the code, so maybe I looked at the wrong place :-)

Reply:
right, copy paste error....